quisp/modules/BellStateAnalyzer.cc

```diff
@@ -29,8 +30,14 @@ class BellStateAnalyzer : public cSimpleModule {
  private:
   // for performance analysis
   int n_res = 0;
+  int trials = 0;
   simsignal_t GOD_num_resSignal;
-
+  std::string BSA_perf_output_filename;
+  std::vector<simtime_t> creation_time;
+  simsignal_t Average_Num_TrialSignal;
```

Comment:
could you use lower_case name for the local variables in your change? 
for example, this should be `average_num_trial_signal`.

see also https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
OK, I fixed it. Thank you.